pkg/pipelines/secrets.go

```diff
@@ -13,10 +13,10 @@ import (
 
 // createOpaqueSecret creates a Kubernetes v1/Secret with the provided name and
 // body, and type Opaque.
-func createOpaqueSecret(name types.NamespacedName, data string) (*corev1.Secret, error) {
+func createOpaqueSecret(name types.NamespacedName, data, secretName string) (*corev1.Secret, error) {
 	r := strings.NewReader(data)
```

Comment:
secretName is actually "key" to the map data structure.   It is not the secretName.   The secretName is name arguement (which is a namespacedName).